assets/wix/Product.wxs

```diff
@@ -84,6 +84,16 @@
                   Execute="deferred"
                   Return="ignore"
                   Impersonate="no" />
+    <SetProperty Id="EnableMU"
+                 Before="EnableMU"
+                 Sequence="execute"
+                 Value="&quot;[VersionFolder]pwsh.exe&quot; -NoProfile -ExecutionPolicy Bypass -Command &quot;$null=(New-Object -ComObject Microsoft.Update.ServiceManager).AddService2('7971f918-a847-4430-9279-4a52d1efe18d', 7, '')&quot;" />
```

Comment:
What if it is first pwsh install (no pwsh on the computer)?

Reply:
This happens after `pwsh` is installed like all the other actions.